cl_billing_pricing.rules_default.inc

```diff
@@ -1,46 +0,0 @@
-<?php
```

Comment:
I think we still want this file -- `commerce_product_calculate_sell_price` is the default way of pricing recurring line items and we want this module to be useful to general Drupal.org users of CLB as well.